README.md

```diff
@@ -5,21 +5,97 @@ Right now this app only works on android. iOS support is planned in the future.
+
+If you have already installed JDK on your system, we recommend JDK 17. You may encounter problems using higher JDK versions. To make sure you are using the JDK version 17 update the `JAVA_HOME` and `PATH` variable to reflect the same.
+
+```sh
+export JAVA_HOME=/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home/
+export PATH=$JAVA_HOME/bin:$PATH
+```
+
+### Setup Android Studio
+
+[Download and Install Android Studio](https://developer.android.com/studio/index.html)
+
+While on Android Studio installation wizard, make sure the boxes next to all of the following items are checked:
+
+- Android SDK
+- Android SDK Platform
+- Android Virtual Device
+
+Then, click "Next" to install all of these components.
+
+Android Studio installs the latest Android SDK by default. Building a React Native app with native code, however, requires the `Android 14 (UpsideDownCake)` SDK in particular. Additional Android SDKs can be installed through the SDK Manager in Android Studio.
+
+To do that, open Android Studio, click on "More Actions" button and select "SDK Manager".
 
-Create a `local.properties` file in `/android` folder and define the sdk path as shown below:
+Select the "SDK Platforms" tab from within the SDK Manager, then check the box next to "Show Package Details" in the bottom right corner. Look for and expand the `Android 14 (UpsideDownCake)` entry, then make sure the following items are checked:
 
+- Android SDK Platform
+- `Intel x86 Atom_64 System Image` or `Google APIs Intel x86 Atom System Image` or (for Apple M1 Silicon) `Google APIs ARM 64 v8a System Image`
+
+You also need to setup your environment variables to use the android utlities just installed. In your `~/.zshrc` or `~/.zprofile` (or `~/.bashrc` file if use bash shell) add the following lines
+
+```sh
+export ANDROID_HOME=$HOME/Library/Android/sdk
+export PATH=$PATH:$ANDROID_HOME/emulator
+export PATH=$PATH:$ANDROID_HOME/platform-tools
+```
+
+then run 
+
+```sh
+source ~/.zshrc
 ```
-sdk.dir=/Users/<USERNAME>/Library/Android/sdk
+
+Verify that `ANDROID_HOME` has been set by running `echo $ANDROID_HOME` and the appropriate directories have been added to your path by running `echo $PATH`.
+
+> Please make sure you use the correct Android SDK path. You can find the actual location of the SDK in the Android Studio "Settings" dialog, under Languages & Frameworks → Android SDK.
+
+### Clone the project
+
+```sh
+git clone https://github.com/Blockchain-Powered-eSIM/LPA.git
 ```
 
-Replace the `<USERNAME>` with your device username
+This project uses React Native's built-in command line interface. Rather than install and manage a specific version of the CLI globally, we recommend you access the current version at runtime using `npx`, which ships with Node.js. With `npx react-native <command>`, the current stable version of the CLI will be downloaded and executed at the time the command is run.
```

Comment:
This project uses React Native's built-in command line interface. Rather than installing and managing a specific version of the CLI globally, we recommend accessing the current version at runtime using `npx`, which ships with Node.js. With `npx react-native <command>`, the current stable version of the CLI will be downloaded and executed when the command is run.